src/views/Library.vue

```diff
@@ -12,11 +12,18 @@
     </VRow>
     <VRow v-if="rescan">
       <VBtn @click="start" color="success" class="ma-2">
-        <VIcon left>mdi-refresh {{ rescan.running ? "mdi-spin" : "" }}</VIcon>
+        <VIcon left
+          >mdi-refresh
+          {{ rescan.last_click > rescan.finished_at ? "mdi-spin" : "" }}</VIcon
+        >
         {{ $t("library.start-scan") }}
       </VBtn>
     </VRow>
     <VRow class="flex-column" v-if="rescan">
+      <div>
+        <strong>{{ $t("library.last-scan") }}: </strong>
+        {{ new Date(rescan.finished_at).toLocaleString() }}
```

Comment:
Use [vue-i18n's builtin datetime formatting](https://kazupon.github.io/vue-i18n/guide/datetime.html) here. The locale setting in the app might differ from the browser's.

Reply:
have done this. The way vue-i18n handles dates isn't optimal though, it decides too much for us

Reply:
In what way does it decide to much for us?

Reply:
Probably because we can't do this? https://github.com/kazupon/vue-i18n/issues/625

Reply:
there seems to be little control in the order. For `en` this is `Sat, Mar 21, 2020, 4:59 PM ` even though the format is defined in the order year, month, day, ...
There is also no option to define whenther to use a comma, slash, ... in the notation

Reply:
> Probably because we can't do this? [kazupon/vue-i18n#625](https://github.com/kazupon/vue-i18n/issues/625)

indeed. Though if this is in the works, I would keep the current formatting it uses.

Reply:
Yeah, I see what you mean. I do think the benefit of using our i18n library outweighs the disadvantage of having little control over the formatting.

I subscribed to (and +1'd) the issue I linked, so if it ever gets implemented we can change it to be more to our liking.

Reply:
👍